jishaku/features/management.py

```diff
@@ -212,38 +212,50 @@ async def jsk_sync(self, ctx: ContextA, *targets: str):
         paginator = commands.Paginator(prefix='', suffix='')
 
         guilds_set: typing.Set[typing.Optional[int]] = set()
+        clear_guilds_set: typing.Set[typing.Optional[int]] = set()
         for target in targets:
-            if target == '$':
-                guilds_set.add(None)
+            active_set = guilds_set
+            if target[0] == '-':
+                active_set = clear_guilds_set
+                target = target[1:]
+            if target in ('', '$'):
+                active_set.add(None)
```

Comment:
I feel like there's gotta be a cleaner way of doing this. The plain method of parsing was probably fine when it was just a few special cases, but maybe it needs a bit of a do-over if we want it to handle removals and transferring of commands. Not sure of the approach though.